app-text/zathura/zathura-0.5.11.ebuild

```diff
@@ -1,44 +1,43 @@
-# Copyright 1999-2024 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
 inherit meson virtualx xdg
 
-DESCRIPTION="A highly customizable and functional document viewer"
+DESCRIPTION="Highly customizable & functional document viewer"
 HOMEPAGE="https://pwmt.org/projects/zathura/"
 
 if [[ ${PV} == *9999 ]]; then
 	inherit git-r3
 	EGIT_REPO_URI="https://git.pwmt.org/pwmt/${PN}.git"
 	EGIT_BRANCH="develop"
 else
-	SRC_URI="
-		https://github.com/pwmt/zathura/archive/${PV}.tar.gz -> ${P}.tar.gz
-	"
+	SRC_URI="https://github.com/pwmt/zathura/archive/${PV}.tar.gz -> ${P}.tar.gz"
 	KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86 ~amd64-linux ~x86-linux"
 fi
 
 LICENSE="ZLIB"
-SLOT="0/5.6"
-IUSE="man seccomp synctex test"
-
+SLOT="0/6.7" # plugin versions api.abi (see meson.build)
+IUSE="+man seccomp synctex test"
```

Comment:
https://github.com/search?q=repo%3Apwmt%2Fzathura+GDK_WINDOWING&type=code

Can you adapt the useflags and src_configure to fix that bug? https://bugs.gentoo.org/958021

See the context: https://bugs.gentoo.org/624960
https://public-inbox.gentoo.org/gentoo-dev/20240623173646.3368935-1-eschwartz93@gmail.com/